dcos/http.py

```diff
@@ -100,7 +100,8 @@ def _request(method,
                "please run: `dcos config set core.ssl_verify <value>`")
         description = config.get_property_description("core", "ssl_verify")
         if description is not None:
-            msg += "\n<value>: {}".format(description)
+            msg += "\n\ncore.ssl_verify: {}".format(description)
```

Comment:
Lets keep this line the same. We print out `"please run: `dcos config set core.ssl_verify <value>`` right before this, so it makes sense to explain what "<value>" is.